frontend/src/scenes/billing/v2/Billing.tsx

```diff
@@ -312,21 +312,23 @@ const BillingProduct = ({ product }: { product: BillingProductV2Type }): JSX.Ele
         700: 'medium',
     })
 
+    const showFreeAllocationLimit = !billing?.has_active_subscription && product.free_allocation !== null
+
     const billingGaugeItems: BillingGaugeProps['items'] = useMemo(
         () =>
             [
                 {
                     tooltip: (
                         <>
                             <b>Free tier limit</b>
```

Comment:
Why don't we merge the free tiers here? (since we said subscribed and unsubscribed should have the same free allocation

Reply:
I figured leave it for now. If we all end up fully in agreement that we don't want to use the free allocation then I will come back and remove all reference to it